docs/api/instruments/agilent/agilent8648.rst

```diff
@@ -0,0 +1,7 @@
+##############################################
+Agilent 8648 Signal Generator
+##############################################
+
+.. automodule:: pymeasure.instruments.agilent.agilent8648
```

Comment:
Please add each instrument class you implemented here

Reply:
Ok